docs/src/components/sender.md

```
@@ -121,6 +121,26 @@ Sender 组件支持在多行模式下灵活定制底部区域。通过 `footer-l
 
 结合 `buttonGroup` 属性，您可以实现更复杂的交互逻辑。例如，通过监听 `files-selected` 事件返回的文件列表，动态地禁用上传按钮或提交按钮，并更新其 `tooltips` 提示信息，以引导用户操作。
 
+**自定义 Tooltip 位置**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)